debian/changelog

```diff
@@ -1,4 +1,7 @@
   [Graeme Weatherill]
+  * Adds 'hypo_loc' slot to RuptureContext object
+  
+  [Graeme Weatherill]
```

Comment:
You can keep together all the feature you implemented if they are chronologically consecutive.

I mean
```
[Graeme Weatherill]
feature 1
feature 2
```

and not

```
[Graeme Weatherill]
feature 1

[Graeme Weatherill]
feature 2
```

If (for instance) Marco Pagani does something which chronologically enters between `feature 1` and `feature 2` then you can repeat your name.